src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -2484,7 +2491,96 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/calculateSize':
+    post:
+      tags:
+        - nodes
+      summary: Calculate a folder size
+      description: |
+        
+        This may be executed against a node specified by **nodeId** and this nodeId must be a valid node ID.
+        Furthermore, this POST endpoint will be executed **asynchronously** with a `202` HTTP response signifying that
+        the request has been accepted successfully.
+        
+        The response body will include the ```nodeId``` of the pending action, which can be used in a
+        **GET/calculateSize** endpoint to check if the action's status has been completed, at which point the result will be returned, comprising ```the size of the node in bytes```.
+        
+        Essentially, ```maxItems``` ***parameter*** is utilized for pagination purposes.
+        
+        By default, **results are limited to the first 100.**
+        Results can be restricted using "paging". For example:
+        
+        ```JSON
+        "paging": {
+          "maxItems": "50",
+          "skipCount": "28"
+        }
+        ```
+        This assists in handling scenarios where large files or subfolders exist within a folder, allowing the folder size determination mechanism to function in chunks.
 
+      operationId: calculateSize
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/nodeTargetIdParam'
+        - $ref: '#/parameters/maxItemsForSizeParam'
+      responses:
+        '202':
+          description: Request Accepted
+          schema:
+            $ref: '#/definitions/sizeAcknowledgedResponse'
+        '400':
+          description: |
+            Invalid parameter: value of **nodeId** is invalid
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission for **nodeId**
+        '404':
+          description: Entity with **nodeId** does not exist
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    get:
+      tags:
+        - nodes
+      summary: Get the size of folder after initiating POST/calculateSize Endpoint.
+      description: |
+        ***Receiving the size***
+        
+        This endpoint takes ```nodeId``` as a path parameter and returns the result after executing above POST request.
+        
+        Below are the three stages, which reflect the status for the present execution ***nodeId***.
+        
+        ```
+           NOT-INITIATED: Not Initiated yet.
```

Comment:
Is it possible to receive this?

Reply:
Yes

Reply:
I don't see how it is possible? Doesn't the job start immediately when the user makes the POST request?

Reply:
Please document the answer

Reply:
Previously, we configured the nodeService property attribute to handle all three stages of POST activity.
However, now we will look at the implementation with an **execution job ID** instead of nodeId and will return the **execution job ID**  in response.

Reply:
Now, we are returning **executionJob** Id and maintaining all stages that correspond to that node ID.